config/conversion_issues.json

```diff
@@ -180,6 +180,23 @@
         "a": "more info"
       }
     },
+    {
+      "key": "lessons_attached_content",
+      "active": true,
+      "tool": "sakai.lessonbuildertool",
+      "icon-class": "icon-edit",
+      "description": "Lessons attached content",
+      "detail-heading": "Lessons pages",
+      "found": "Some Lessons pages have attachments that could not be moved automatically. These have been identified and placeholder put in place. Review and update this content.",
```

Comment:
embedded not attachments, but needs a clearer description. @TurRil perhaps Thomas can suggest better wording.